### PR TITLE
Add ADOPTERS file, per CNCF requirements

### PR DESCRIPTION
**Pull request by technosophos**
This adds an ADOPTERS.md file, which CNCF requires for graduating from Incubation.

Signed-off-by: Matt Butcher <matt.butcher@microsoft.com>

### CODE REVIEW

**Comment by technosophos** [comment]
If you would like to add your org before this PR gets merged, use the "Suggest" feature in GitHub Comments, and I will merge the suggestions.

Example:

Suggest

File: ADOPTERS.md
Comment (jdolitsky) [inline ADOPTERS.md]:
```suggestion

- [Blood Orange](https://bloodorange.io)
```

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
Needs a DCO sign off before merging but lgtm!

File: ADOPTERS.md
Comment (deimosfr) [inline ADOPTERS.md]: - [Qovery](https://www.qovery.com)

**Review by deimosfr — COMMENTED** [review]
Adding Qovery to the list

File: ADOPTERS.md
Comment (hickeyma) [inline ADOPTERS.md]:
```suggestion
[IBM](https://www.ibm.com)
```

Comment (mattfarina) [reply ADOPTERS.md]: You can also add Samsung SDS...

```
[Samsung SDS](https://www.samsungsds.com/)
```

**Comment by technosophos** [comment]
@bacongobbler it appears that suggestions don't always get --signoff set (Not sure how Martin's worked but Josh's didn't). Any idea how to fix that?

**Comment by bacongobbler** [comment]
Sorry no I haven't found a good way to use suggestions. I usually go back and `git rebase -i`, marking it as a re-word to add the sign-off.

File: ADOPTERS.md
Comment (marckhouzam) [inline ADOPTERS.md]:
```suggestion
[Ville de Montreal](https://montreal.ca)
```

Comment (marckhouzam) [reply ADOPTERS.md]: I tried the chrome extension to add my sign-off, but I don't believe it works for suggestions.
Here is the sign off:
Signed-off-by: Marc Khouzam <marc.khouzam@montreal.ca>

**Comment by marckhouzam** [comment]
For the sign-off, I got the impression it can be added in a text box just before committing the suggestion.  As long as the committer knows what the sign-off line is.  I haven't tried though.

**Comment by technosophos** [comment]
Okay, fixed the sign-offs

File: ADOPTERS.md
Comment (marckhouzam) [inline ADOPTERS.md]: My bad, forgot the intial dash:
```suggestion
- [Ville de Montreal](https://montreal.ca)
```
Signed-off-by: Marc Khouzam <marc.khouzam@montreal.ca>

Comment (marckhouzam) [reply ADOPTERS.md]: I submitted #7555 for this.

**Pull request merged**
